tools/cmake/preset/zephyr.cmake

```diff
@@ -0,0 +1,34 @@
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set_overridable_option(CMAKE_TOOLCHAIN_FILE                     "${PROJECT_SOURCE_DIR}/examples/arm/ethos-u-setup/arm-zephyr-eabi-gcc.cmake")
```

Comment:
Can you verify this works? I'm pretty sure setting `CMAKE_TOOLCHAIN_FILE` within cmake itself actually doesn't work. You need to set this in the CMakePreset.json (see the iOS/Apple examples)